api/views.py

```diff
@@ -1,7 +1,23 @@
 from django.shortcuts import render
+from django.http import HttpResponse, JsonResponse
+from rest_framework.parsers import JSONParser
+from lyrics.models import Lyric
+from .models import LyricSerializer
 
 
 def test(request):
     template = ""
     context = {}
     return render(request=request, template_name=template, context=context)
+
+
+@api_view(["GET"])
+def lyric_detail(request, pk):
+    try:
+        lyric = Lyric.objects.get(pk=pk)
+    except Lyric.DoesNotExist:
+        return HttpResponse(status=404)
+
+    if request.method == "GET":
+        serializer = LyricSerializer(lyric)
+        return JsonResponse(serializer.data)

```

Comment:
hmmm, You're supposed to leverage `djangorestframework's` `Response` class. Also, I remember also saying that on rule of thumb or **code best practice** to follow is: **explicit is better than implicit**, so in all, that line of code should be: `Response(data=serializer.data, status=status.HTTP_200_OK)

Reply:
**Explicit is better than implicit**: https://www.python.org/dev/peps/pep-0020/. After you're done, check that out. It's called the **Zen of python**.

Reply:
is this done?

Reply:
yeah